pkgs/data/fonts/comic-neue/default.nix

```diff
@@ -1,20 +1,27 @@
-{stdenv, fetchurl, unzip}:
+{ stdenv, fetchurl
+, unzip }:
 
 stdenv.mkDerivation rec {
-  name = "comic-neue-1.1";
+  name = "comic-neue-${version}";
+  version = "2.2";
 
   src = fetchurl {
-    url = "http://comicneue.com/comic-neue-1.1.zip";
-    sha256 = "f9442fc42252db62ea788bd0247ae0e74571678d1dbd3e3edc229389050d6923";
+    url = "http://comicneue.com/${name}.zip";
+    sha256 = "1dmmjhxxc0bj2755yksiiwh275vmnyciknr9b995lmdkjgh7sz6n";
   };
 
-  buildInputs = [unzip];
+  buildInputs = [ unzip ];
   phases = [ "unpackPhase" "installPhase" ];
   sourceRoot = name;
 
   installPhase = ''
-    mkdir -p $out/share/fonts/truetype
-    cp -v *.ttf $out/share/fonts/truetype
+    mkdir -vp $out/share/fonts/truetype $out/share/fonts/OTF $out/share/fonts/EOT $out/share/fonts/WOFF $out/share/fonts/WOFF2 $out/share/doc/${name}
+    cp -v OTF/*.otf  $out/share/fonts/OTF
```

Comment:
Several other packages use `$out/share/fonts/opentype`, better to follow that convention.